tests/docker-compose.yml

```diff
@@ -1,3 +1,5 @@
+---
+
```

Comment:
why is this needed ?

Reply:
a result of my yarn linter 😅 !

Reply:
Hmmm that is only needed if you want to describe several YAML objects in the same file, such as for a kubernetes manifest... It's not needed here, so I'd keep the file as is

Reply:
Thanks, did knew it 👍🏾